base_location/README.rst

```diff
@@ -12,24 +12,48 @@ It enables zip, city, state and country auto-completion on partners and companie
 
 Also allows different search filters.
 
+Configuration
+=============
+
+* Users in group 'Contact Creation' can maintain the Locations.
+
 Usage
 =====
 
+#. Activate the developer mode in *Settings*.
```

Comment:
This should be in Configuration, and in the Usage section, explain that going to a partner, and filling the field "Autocomplete...", they will get the rest of the fields...